main.go

```diff
@@ -73,9 +81,22 @@ func main() {
 	flag.BoolVar(&enableLeaderElection, "leader-elect", false,
 		"Enable leader election for controller manager. "+
 			"Enabling this will ensure there is only one active controller manager.")
-	opts := zap.Options{
-		Development: true,
+
+	opts = zap.Options{
+		ZapOpts: []uberzap.Option{
+			uberzap.AddCaller(),
```

Comment:
I'm trying to understand the need for uberzap here.  The [operator-sdk ](https://sdk.operatorframework.io/docs/building-operators/golang/references/logging/#default-zap-logger)suggests using `--zap-devel` which have the defaults that we need.

Reply:
The `--zap-devel` flag will enable the stacktrace for all levels WARN and above. We don't want that; I couldn't find a Zap flag that equates to {show caller in the logs, don't show stack trace unless it is panic level}, that is the reason I had to craft a ZapOpts object.